src/Installers/Windows/AspNetCoreModule-Setup/IIS-Common/build/versions.props

```diff
@@ -0,0 +1,38 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003"> 
+  <!-- Allow versions as msbuild arguments -->
+
+  <PropertyGroup>
+    <PRODUCT_MAJOR Condition="$(PRODUCT_MAJOR) == ''">7</PRODUCT_MAJOR>
```

Comment:
Can we use the version.props file in the repo root to manage the version of this component?

Reply:
I think this file is meant to be part of the root. IIS-Common doesn't produce any files that require a version.